apps/web-giddh/src/app/vat-report/vat-report-filters/vat-report-filters.component.ts

```diff
@@ -297,9 +376,21 @@ export class VatReportFiltersComponent implements OnInit {
             this.to = dayjs(value.endDate).format(GIDDH_DATE_FORMAT);
             this.toDate.emit(this.to);
             this.getVatReport();
+            this.clearQueryParams();
         }
     }
 
+    /**
+     * Clear query params,
+     * Navigate to the current URL path without query parameters, 
+     *
+     * @private
+     * @memberof VatReportFiltersComponent
+     */
+    private clearQueryParams(): void {
+        this.router.navigate([], { queryParams: {} });
```

Comment:
confirm with @ravindersinghdang sir for this logic

Reply:
Removed